addons/account/models/company.py

```diff
@@ -358,6 +359,13 @@ def _compute_domestic_fiscal_position_id(self):
             potential_domestic_fps = company.fiscal_position_ids.filtered_domain([('country_id', '=', company.country_id.id)]).sorted('sequence')
             company.domestic_fiscal_position_id = potential_domestic_fps[0] if potential_domestic_fps else False
 
+    @api.depends('account_fiscal_country_id')
+    def _compute_account_fiscal_country_group_codes(self):
+        for company in self:
+            company.account_fiscal_country_group_codes = (
+                company.account_fiscal_country_id.country_group_codes if company.account_fiscal_country_id else ['']
```

Comment:
Why this empty string?
```suggestion
                company.account_fiscal_country_id.country_group_codes if company.account_fiscal_country_id else []
```

Reply:
This looks like a bug in the `fields.Json`'s `_convert_to_*` functions.

A Json is not necessary a scalar value. We can't downcast all falsy values to NULL/False.

// cc @odoo/rd-framework-py